pkgs/games/keen4/builder.sh

```diff
@@ -1,10 +1,15 @@
-source $stdenv/setup
```

Comment:
builder.sh is really deprecated and very out of date. Instead of improving the bash here it would be a better idea to convert it to phases to bring it up to today standards.

Reply:
I've already spent the time to improve the current implementation, though. This change shouldn't make it any harder to convert to phases.

Reply:
It actually makes it harder to convert because half of the changes would be reverted.